adala/runtimes/_litellm.py

```diff
@@ -146,55 +132,6 @@ def client(self):
             mode=instructor.Mode(self.instructor_mode),
         )
 
-    def init_runtime(self) -> "Runtime":
-        # check model availability
-        # extension of litellm.check_valid_key for non-openai deployments
-        try:
-            self._check_client()
```

Comment:
if we are not doing this here at all anymore, where are we relying on making these checks, and are we confident nothing can change by the time we get here (running inference)?

Reply:
I think the answer is that we're not making these checks automatically at all anymore, we're counting on calling validate-connection from LSE close enough to other calls that nothing changes

Reply:
Personally I'm on board with this (as long as it passes rigorous testing) because it'll make debugging and local dev in the adala app easier. Currently if we instantiate an agent wrong or lose connection, it raises an error deep in pydantic library code before it even gets to the endpoint handler, which is hard to deal with.